scripts/operations/node_management/Add_River_Cabinets/verify_bmc_credentials.sh

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env bash
+# MIT License
+#
+# (C) Copyright 2022 Hewlett Packard Enterprise Development LP
+#
+# Permission is hereby granted, free of charge, to any person obtaining a
+# copy of this software and associated documentation files (the "Software"),
+# to deal in the Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish, distribute, sublicense,
+# and/or sell copies of the Software, and to permit persons to whom the
+# Software is furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included
+# in all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+
+set -u
+
+# For each redfish endpoint
+for endpoint in $(cray hsm inventory redfishEndpoints list --laststatus HTTPsGetFailed --format json | jq .RedfishEndpoints[].ID -r); do
+    echo
+    echo "------------------------------------------------------------"
+    echo "Redfish Endpoint $endpoint has discovery state HTTPsGetFailed"
+    # Ignore the failure if it is a master node    
+    # cray sls search hardware list --parent $BMC --format json | jq '.[].ExtraProperties.Role' -r
```

Comment:
If this is commented, why check it in?
```suggestion
```

Reply:
Applied here: https://github.com/Cray-HPE/docs-csm/pull/2344/commits/ab3c45572fa5b86e6e61bbd196c6734919b6dae8#diff-c2621ddd43f3ea34a8b4868793aa0716caa9cbce58a64280ccc3c94784b9b4f2L32